src/main/java/com/radixdlt/constraintmachine/PermissionLevel.java

```diff
@@ -0,0 +1,11 @@
+package com.radixdlt.constraintmachine;
+
+/**
+ * Ordered Execution level which specifies what level of permission the
+ * constraint machine is running on. Depending on this level, some particle
+ * transitions may be allowed or not.
+ */
+public enum PermissionLevel {
```

Comment:
@talekhinezh what's that for?

Reply:
to be able to allow/not-allow commands coming from either a user or the consensus system (e.g. updating the current timestamp of the system)